### PR TITLE
hostapd: radius server support for WPA-PSK

### PR DESCRIPTION
**Pull request by dsinghra123**
Hostapd provides Radius Server support for WPA-PSK but hostapd.sh
lacks this config option for WPA_PSK. Also, there is no config
option for radius server support for MAC Based authentication.
This patch includes both cases.

Signed-off-by: Deependra Rawat deependra.rawat85@gmail.com

### CODE REVIEW

**Comment by adrianschmutzler** [comment]
@dsinghra123 This needs to be rebased.

**Comment by adrianschmutzler** [comment]
Hi, since this is not a fix, but a feature, and none of the committer seems to be interested in reviewing it, I will close it now (i.e. "Won't implement").

I'm sorry, but better crush your remaining hopes now than waiting another year where supposedly nothing different will happen.

File: package/network/services/hostapd/files/hostapd.sh [hunk 1]
```diff
@@ -574,6 +596,11 @@ hostapd_set_bss_options() {
 			append bss_conf "macaddr_acl=0" "$N"
 			append bss_conf "deny_mac_file=$_macfile" "$N"
 		;;
+		radius)
+			append bss_conf "macaddr_acl=2" "$N"
+			# radius server can be used to assign dynamic VLAN based on MAC
+			vlan_possible=1
+		;;
```
Comment (pali) [inline package/network/services/hostapd/files/hostapd.sh:603]: You should clear there `_macfile` variable, like few lines below for `*)`... `_macfile=""`

**Comment by pali** [comment]
Any chance for this usefull feature in openwrt? I proposed very similar pull request too: https://github.com/openwrt/openwrt/pull/2971

**Comment by DhimantThanki** [comment]
Hi,

Can you share the wireless config file to get more idea what other parameters needed to get the SSID working ?

**Pull request closed without merging**